rest-api-spec/src/main/resources/rest-api-spec/api/indices.disk_usage.json

```
@@ -0,0 +1,61 @@
+{
+  "indices.disk_usage": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-disk-usage.html",
+      "description": "Analyzes the disk usage of each field of an index"
+    },
+    "stability": "beta",
```

I think this needs to be `experimental` to be consistent with the docs.
```suggestion
    "stability": "experimental",
```